bnelson777/id.ly
# IL-95: Add keyboard spacer to create card screen

Could not add the KeyboardSpacer component we are already using due to problems centering on fields in this screen. Added new library KeyboardAwareScrollView to fix the problem.

-Choosing an input field in create card screen now centers keyboard on field

Testing Procedure
Before: Choosing field in card create screen was not reactive, would not center on keyboard. As a result views would be obscured.
1. Go to create card screen through wallet.
2. Add info to each field.
3. Add several attributes and fill the fields with text.
4. Verify keyboard centers on new attributes as they are being added.
5. Verify all text fields center on keyboard when selected.
